include/flashinfer/norm.cuh

```diff
@@ -133,6 +133,8 @@ __global__ void FusedAddRMSNormKernel(T* __restrict__ input, T* __restrict__ res
     input_vec.fill(0.f);
     vec_t<T, VEC_SIZE> residual_vec;
     residual_vec.fill(0.f);
+    vec_t<float, VEC_SIZE> x_vec;
```

Comment:
I wrote this kernel in August https://github.com/flashinfer-ai/flashinfer/pull/419, and you can actually use https://pytorch.org/docs/stable/benchmark_utils.html to add a benchmark. This way, you can know whether there is a performance improvement before and after the changes.

Reply:
https://triton-lang.org/main/python-api/generated/triton.testing.Benchmark.html is also ok

Reply:
> I wrote this kernel in August #419, and you can actually use https://pytorch.org/docs/stable/benchmark_utils.html to add a benchmark. This way, you can know whether there is a performance improvement before and after the changes.

Okay, I'll look into this, but I've analyzed this PR using Nsign Compute and found that the performance is about the same as the code before the precision improvement(#587).

Reply:
Something like this file would be great
https://github.com/flashinfer-ai/flashinfer/tree/main/benchmarks

Reply:
Okay,I'll try to write a benchmark test like this.